src/test/java/com/sumologic/http/sender/SumoHttpSenderTest.java

```diff
@@ -39,7 +39,7 @@
 
 public class SumoHttpSenderTest {
 
-    private static final int PORT = 10010;
+    private static final int PORT = 26932;
```

Comment:
this was needed because port 10010 is actually reserved for Docker daemon on Travis CI hosts.